frontend/src/hooks/queryBuilder/useGetAggregateKeys.ts

```diff
@@ -11,24 +11,30 @@ type UseGetAttributeKeys = (
 	options?: UseQueryOptions<
 		SuccessResponse<IQueryAutocompleteResponse> | ErrorResponse
 	>,
+	isInfraMonitoring?: boolean,
 ) => UseQueryResult<
 	SuccessResponse<IQueryAutocompleteResponse> | ErrorResponse
 >;
 
 export const useGetAggregateKeys: UseGetAttributeKeys = (
 	requestData,
 	options,
+	isInfraMonitoring,
 ) => {
 	const queryKey = useMemo(() => {
 		if (options?.queryKey && Array.isArray(options.queryKey)) {
-			return [QueryBuilderKeys.GET_AGGREGATE_KEYS, ...options.queryKey];
+			return [
+				QueryBuilderKeys.GET_AGGREGATE_KEYS,
+				...options.queryKey,
```

Comment:
we have changes here as well and we have another `useGetAggregateKeys` in `infraMonitoring` hooks as well  ?

Reply:
@vikrantgupta25 Updated the code for this.